src/server/mappers/UrlMapper.ts

```diff
@@ -14,11 +14,21 @@ export class UrlMapper implements Mapper<StorableUrl, UrlType> {
     const { UrlClicks: urlClicks } = urlType
     if (!urlClicks || !Number.isInteger(urlClicks.clicks))
       throw new Error('UrlClicks object not populated.')
+
+    const tagStrings: string[] = []
+    const { tags } = urlType
+    if (tags) {
+      for (let i = 0; i < tags.length; i += 1) {
+        tagStrings.push(tags[i].tagString)
+      }
+    }
```

Comment:
Small TS-related suggestion, might want to simplify it with the `.map()` function.

Reply:
I fixed this